include/mgp.hpp

```diff
@@ -653,7 +657,7 @@ class Relationship {
   class Properties Properties() const;
 
   /// @brief Returns the value of the relationship’s `property_name` property.
-  Value operator[](const std::string_view property_name) const;
+  Value operator[](const std::string &property_name) const;
```

Comment:
Is `Value &operator[](const std::string &property_name)` missing in class Relationship?

Reply:
Done